website/content/docs/auth/saml.mdx

```diff
@@ -0,0 +1,158 @@
+---
+layout: docs
+page_title: SAML - Auth Methods
+description: |-
+  The "saml" auth method allows users to authenticate with Vault using their
+  identity in a SAML Identity Provider.
+---
+
+# SAML Auth Method
+
+<EnterpriseAlert />
+
+The `saml` auth method allows users to authentication with Vault using their identity
+within a [SAML V2.0](https://saml.xml.org/saml-specifications) Identity Provider.
+Authentication is suited for human users by requiring interaction with a web browser.
+
+## Authentication
+
+### Via the UI
+
+1. Select "SAML" from the "Method" select box.
+1. Enter a role name into the "Role" input field or leave blank to use
+   the [default role](/vault/api-docs/auth/saml#default_role).
+1. Press "Sign in with SAML Provider" and complete the authentication with the
+   configured SAML Identity Provider.
+
+### Via the CLI
+
+The CLI login defaults to the path of `/saml`. If this auth method was enabled at a
+different path, specify `-path=/my-path` in the CLI.
+
+```shell-session
+$ vault login -method=saml role=admin
+
+Complete the login via your SAML provider. Launching browser to:
+
+    https://company.okta.com/app/vault/abc123eb9xnIfzlaf697/sso/saml?SAMLRequest=fJI9b9swEIZ3%2FwqBu0SJ%2FpBDRAZce4iBtDViN0MX40Sda...
+```
+
+The default browser will open to the generated URL where the user must authenticate with
+the configured SAML Identity Provider. The URL may be manually entered into the browser
+if it cannot be automatically opened.
+
+The CLI login behavior may be customized with the following optional parameters:
+
+- `skip_browser` (default: `false`): If set to `true`, automatic launching of the default
+  browser will be skipped. The SAML Identity Provider URL must be manually entered in a
+  browser to complete the authentication flow.
+- `abort_on_error` (default: `false`): If set to `true`, failure to launch the default
+  browser will result in a non-zero exit of the CLI and an error displayed.
+
+## Configuration
```

Comment:
Follow-up question: from context, I'm guessing that the "Configuration" section is meant to explain how folks can set up SAML authN and the "Authentication" section is meant to explain how Vault users would subsequently use SAML to authenticate, is that right?

If so, I'd recommend swapping the order of the sections so we're telling folks how to set up the authentication method before talking about how to use it.

Reply:
> Follow-up question: from context, I'm guessing that the "Configuration" section is meant to explain how folks can set up SAML authN and the "Authentication" section is meant to explain how Vault users would subsequently use SAML to authenticate, is that right?

That's correct! I followed this ordering because it is most consistent with the rest of the Vault auth method docs. My understanding is that these docs aren't supposed to read as step-by-step tutorials. I believe that the "Authentication" section comes first because a majority of Vault users are not the admin persona. The admin persona is interested in the "Configuration" section.

Would it be okay to leave the section headings and ordering to be consistent with the rest of the auth method docs?